packages/components/jest.config.js

```diff
@@ -0,0 +1,6 @@
+const defaults = require('../../node_modules/@talend/scripts-config-jest/jest.config.js');
+
+module.exports = {
+	...defaults,
+	setupFilesAfterEnv: [...defaults.setupFilesAfterEnv, '../../test-setup.js'],
```

Comment:
This custom jest config was added because some tests rely on a date mock that is set in the talend/ui > test-env.js.
Don't know if it makes sense to add that to talend/scripts > jest config. Opinions welcome

Reply:
Hum it seems to be only used in 4 test files, and 2 times the same 2 files 😅 
So I think it can be done directly in files, copying something like that only one time does not seems to be a big deal for me 🤔

Reply:
That's ok for me, let's do that :)